pycbc/injcutter.py

```diff
@@ -0,0 +1,318 @@
+# Copyright (C) 2013 Ian Harry
```

Comment:
Only if the package is called pycbc.inject and not pycbc.injection as stated above.